thorlcr/master/thmastermain.cpp

```diff
@@ -1055,7 +1058,7 @@ int main( int argc, const char *argv[]  )
                 throw makeStringException(TE_AbortException, "Failed to apply worker networkpolicy manifest");
             k8s::KeepJobs keepJob = k8s::translateKeepJobs(globals->queryProp("@keepJobs"));
 
-            std::list<std::pair<std::string, std::string>> params = { { "graphName", graphName}, { "master", myEp.str() }, { "_HPCC_NUM_WORKERS_", std::to_string(numWorkers/numWorkersPerPod)} };
+            std::list<std::pair<std::string, std::string>> params = { { "graphName", graphName}, { "master", myEp.str() }, { "_HPCC_NUM_WORKERS_", std::to_string(numWorkers/numWorkersPerPod)}, { "graphNo", std::to_string(currentGraphNumber) } };
```

Comment:
Harmless, can keep, but I don't think it is strictly necessary to add the instance number to the args of the worker job here? (nothing uses it?)